.github/workflows/release.yaml

```diff
@@ -0,0 +1,165 @@
+name: release
+
+on:
```

Comment:
This workflow is identical to the build.yaml workflow that's been removed, the only change is that we're not running this on pull requests.

Reply:
We can probably delete the conditionals here on if it's a push.